### ISSUE TITLE
collection runner: filter failed

### ISSUE THREAD

**Issue by adampopkiewicz** [a85/POSTMan-Chrome-Extension#1853]
tldr; please add optional filter on results to just show requests that failed.

I have big collection file, it's useful to run all tests in single go to verify if everything is ok with api. When few of them fail, it'd be nice to be able to just focus on these failed ones. For that I always switch to newman, but it's not automatically in sync with my collection/team, so I'd prefer to use collection runner.

in newman you have breakdown of test results over folders which helps a lot, so there are few ways to approach this issue. also it can be useful to have progress indication during running requests based on current request and total number of requests.

**Comment by rmabraham** [comment]
Any update on when this will be added? This makes using the runner so much more efficient.

**Comment by sdnts** [comment]
We've been working on a redesign of the Runner that addresses your issue.

The new Runner is on our Canary channel, and you can download it for [OSX](https://dl.pstmn.io/download/channel/canary/osx_64) / Windows ([x86](https://dl.pstmn.io/download/channel/canary/windows_32) or [x64](https://dl.pstmn.io/download/channel/canary/windows_64)) / Linux ([x86](https://dl.pstmn.io/download/channel/canary/linux_32) or [x64](https://dl.pstmn.io/download/channel/canary/linux_32)) / [Chrome](https://chrome.google.com/webstore/detail/postman/loddepljkabcghihdkbkfknabkinanff)

I'd recommend you to join our Slack community [here](https://www.getpostman.com/slack-invite) so you can provide feedback before the runner moves to our stable builds :)

**Comment by rmabraham** [comment]
Thanks for the quick response. I tested the Canary channel and its look great! 

However whenever I include a datafile the tests don't run it just hangs.

**Comment by sdnts** [comment]
@rmabraham Are you on the Chrome Canary? If so, I'm aware of the issue, and we'll be releasing a fix for it today.

**Comment by rmabraham** [comment]
Yes on the chrome canary.

**Comment by sdnts** [comment]
@rmabraham @adampopkiewicz The new Runner is out with our latest release (4.9) on our native apps. Filtering only failed requests is possible now :)

**Comment by rmabraham** [comment]
@madebysid I tried with the latest release and the runner still looks the same.
![image](https://cloud.githubusercontent.com/assets/10289291/20884570/b0cf5286-baf4-11e6-887a-d7ac894d2029.png)

**Comment by sdnts** [comment]
@rmabraham Looks like you are on the Chrome App. The new Runner makes use of Postman Runtime, so it is out only on the [native apps](http://getpostman.com/apps) as of now.
